ovlb/vue-a11y-combobox
# ⚙️ Add prettier

###  What has been added

Prettier configuration

### How can this be tested

- Change code and see if it conforms to Prettier
- Run linter (e.g. before commit)

### In which browsers have these additions been tested
n/a


### Additonal remarks

n/a
